llvm/lib/Target/RISCV/GISel/RISCVLegalizerInfo.cpp

```diff
@@ -82,7 +82,7 @@ RISCVLegalizerInfo::RISCVLegalizerInfo(const RISCVSubtarget &ST) {
   }
 
   getActionDefinitionsBuilder(G_BSWAP)
-    .clampScalar(0, s16, XLenLLT)
+    .minScalar(0, s16)
```

Comment:
I was suggesting `maxScalar(0, XLenLLT)`. G_BSWAP already requires at s16 so there's no need for minScalar with s16.

Reply:
Updated. Dropped the previous change to show it had no impact on diff.

Reply:
Change to `sXLen`